rvm.sh

```diff
@@ -0,0 +1,924 @@
+#!/usr/bin/env bash
```

Comment:
@learneradarsh, please help me understand, what purpose does the `rvm.sh` file serve in this PR?

I don't think it belongs in this repository.

Reply:
Yeah,But other stuff belongs to this repo only

Reply:
rvm.sh I need to check it